src/bin_editor/Oni2_editor.re

```diff
@@ -58,14 +58,16 @@ let printVersion = _cli => {
   0;
 };
 
+Log.debug("Startup: Parsing CLI options");
```

Comment:
One thing I've seen cause crashes on Windows is logging (writing to `stdout`/`stderr`) before we get a chance to init a console instance (which happens in `Cli.parse`, if `-f` is specified). But I removed this line, and it didn't fix the crash